apps/storefront/src/shared/service/request/fetch.ts

```diff
@@ -1,51 +1,25 @@
-import { snackbar } from '@/utils';
-
-import { RequestType } from './base';
-
 const originFetch = window.fetch;
 
-const responseResult = (path: string, res: any, resolve: any, init: any) => {
-  if (path.includes('current.jwt')) return res.text();
```

Comment:
The only place where we fetch `current.jwt` is here: 

https://github.com/bigcommerce/b2b-buyer-portal/blob/b4c3191363a2a60322ffcae539394e59d7951c31/apps/storefront/src/shared/service/bc/api/login.ts#L12-L13

Which uses a vanilla fetch.